### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Marwa-Sadat**
Screenshot 2024-06-06 074329
1. Display innovative web development projects, with bright colors and modern designs. 🎨✨
2. Highlight the diverse technical and creative skills acquired through several days of dedicated work. 💻🛠
3. Has significant professional achievements and milestones. 🏆📈
4. Includes projects like survey form, Odin project, and tribute page. 📊🔧📄

### CODE REVIEW

**Comment by ahzamir** [comment]
@Marwa-Sadat 

- I can see this view and I don't know what is this:
image

**Comment by Marwa-Sadat** [comment]
@ahzamir  hi sir can you see now please..

**Comment by Marwa-Sadat** [comment]
@ahzamir hi sir please check my project agin.

**Comment by ahzamir** [comment]
@ahzamir 

- Ask your partner for feedback first.

**Comment by Marwa-Sadat** [comment]
@ahzamir hi sir I hope you are well, please check my project, I hope you like it🙂

**Comment by ahzamir** [comment]
@Marwa-Sadat 

- Still i don't know what is this please make your style at least readable.
image

**Comment by shogof** [comment]
@Marwa-Sadat 
### Positives 😊:
1. **Clear Structure**: Your HTML is well-organized, making it easy to maintain. 👍
2. **Your HardWorking**:I like it . 📧

### Areas to Improve 🚀:
1. **Consistent Design**: Aim for consistency in icon styles for a smoother look. 💫
2. **Image Quality**: Consider higher-quality images for better visuals. 🖼️
3. **Enhance Styling**: Experiment with colors and typography for a better appearance. 🎨
4. **Accessibility**: Improve accessibility with alt text for images. 🖼️
5. **Content Completeness**: Replace placeholder text with real content for a more professional touch. 💼

Overall, your site has good points, but there's room for improvement, especially in responsiveness and navbar design! 😊

**Comment by Marwa-Sadat** [comment]
@ahzamir hi sir I hope you are well, please check my project and my partner can't be online from today to tomorrow, please check, I hope you like it.🙂

**Comment by Marwa-Sadat** [comment]
@ahzamir hi sir I hope you are well, please check my project and my partner can't be online from today to tomorrow, please check, I hope you like it.🙂

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        max-width: 720px;
+      }
+    }
+
+    @media (max-width: 992px) {
+      .container {
+        max-width: 960px;
+      }
+    }
+  </style>
```
Comment (ahzamir) [inline index.html:55]: - You have a separated file for your CSS code, move these codes to the `styles.css` file.

**Review by ahzamir — APPROVED** [review]
### Hi @Marwa-Sadat  🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Personal Portfolio Webpage project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html (hunk 1 above)
Comment (Marwa-Sadat) [reply index.html:55]: ok sir

Comment (Marwa-Sadat) [reply index.html:55]: @ahzamir Thank you very much for accepting my project😇🙂🥳😍

**Pull request merged**